server/pbenchinacan/load_keycloak.sh

```diff
@@ -104,3 +145,25 @@ curl -i -X POST "${KEYCLOAK_HOST_PORT}/admin/realms/${REALM}/users/${USER_ID}/ro
   -H "Authorization: Bearer ${ADMIN_TOKEN}" \
   -H "Content-Type: application/json" \
   -d '[{"id":"'${ROLE_ID}'","name":"ADMIN"}]'
+
+# Verify that the user id has a role assigned to it
+USER_ROLES=$(curl -s "${KEYCLOAK_HOST_PORT}/admin/realms/${REALM}/users/${USER_ID}/role-mappings/clients/${CLIENT_ID}" \
+  -H "Authorization: Bearer ${ADMIN_TOKEN}" \
+  -H "Content-Type: application/json" | jq -r '.[].name')
+
+if [[ ${USER_ROLES} == *"ADMIN"* ]]; then
+  echo "The Keycloak configuration is complete."
+else
+  echo "Could not assign client role the user."
+  exit 1
+fi
+# Wait for couple of seconds before we pause and commit the configured container,
+# to make sure all the configurations are properly in place.
+sleep 2
+
+podman pause ${container_name}
+podman commit ${container_name} images.paas.redhat.com/pbench/pbenchinacan-keycloak:${KEYCLOAK_IMAGE_TAG}
+podman unpause ${container_name}
+podman stop ${container_name}
```

Comment:
As discussed in scrum overflow today, pausing the container doesn't guarantee that the Keycloak application state has been flushed to disk, ditto the `sleep`.  It would be better to perform a graceful shutdown of the Keycloak application and then persist the state.

Reply:
Fixed.